pkg/cloudprovider/vsphere/loadbalancer/access.go

```diff
@@ -217,8 +217,8 @@ func (a *access) CreateVirtualServer(clusterName string, objectName types.Namesp
 		Tags:                   a.standardTags.Append(allTags...).Normalize(),
 		DefaultPoolMemberPorts: []string{fmt.Sprintf("%d", mapping.NodePort)},
 		Enabled:                boolptr(true),
-		IpAddress:              ipAddress,
-		ApplicationProfilePath: applicationProfilePath,
+		IpAddress:              strptr(ipAddress),
```

Comment:
Why are these updating to pointers? Guessing they were udpated in the SDK update?

Reply:
Exactly, these are changes in the updated vsphere-automation-sdk-go